src/Microsoft.ML/CSharpApi.cs

```diff
@@ -3456,6 +3481,21 @@ public sealed partial class TrainTestEvaluator
             /// </summary>
             public bool IncludeTrainingMetrics { get; set; } = false;
 
+            /// <summary>
+            /// Column to use for labels
+            /// </summary>
+            public string LabelColumn { get; set; } = "Label";
+
+            /// <summary>
+            /// Column to use for example weight
+            /// </summary>
+            public Microsoft.ML.Runtime.EntryPoints.Optional<string> WeightColumn { get; set; }
+
+            /// <summary>
+            /// Column to use for grouping
+            /// </summary>
+            public Microsoft.ML.Runtime.EntryPoints.Optional<string> GroupColumn { get; set; }
```

Comment:
Is there any default which makes sense here for GorupColumn, such as Label has?

Reply:
The GroupColumn argument has a default value (see in TrainTestMacro.cs). I think this might be an issue with the CSharpCodeGenerator. I'll open an issue.

---
In reply to: [192864697](https://github.com/dotnet/machinelearning/pull/291#discussion_r192864697) [](ancestors = 192864697)